src/pe_reports/report_generator.py

```diff
@@ -353,8 +377,12 @@ def generate_reports(db, datestring, data_directory, output_directory):
             file = filenames[-1]
 
             # Create folders in output directory
-            subprocess.call(["mkdir", f"{output_directory}/ppt"])
-            subprocess.call(["mkdir", f"{output_directory}/{_id}"])
+            try:
+                os.mkdir(f"{output_directory}/ppt")
+                os.mkdir(f"{output_directory}/_id")
+            except FileExistsError as err:
+                logging.error(f"The output directory exists {err}", exc_info=True)
+                return 0
```

Comment:
Is existing output directories a fail state? If so it should return a non-zero value to indicate failure.

Reply:
@mcdonnnj , 

I have updated the code to better handle the error state per commit [0bb568a](https://github.com/cisagov/pe-reports/pull/33/commits/0bb568a663eb003c82d860ba857afaf1c140f77d)

Reply:
Since I was confused here, I wanted to point out that 0bb568a does not address the non-zero return value, but you did fix this in https://github.com/cisagov/pe-reports/pull/33/commits/cb45751df9557cabc8534c2ca9bc79861ab97cf9.